src/Migrator.php

```diff
@@ -16,6 +16,7 @@
 use P4\MasterTheme\Migrations\M012RemoveThemeEditorOption;
 use P4\MasterTheme\Migrations\M013RemoveDuplicatedOptions;
 use P4\MasterTheme\Migrations\M014RemoveDropdownNavigationMenusOption;
+use P4\MasterTheme\Migrations\M015CreateDefaultActionType;
```

Comment:
We just merged a `M015` migration at ee9bfcb1467152c4aa7c0bae411c23f45e563044, so you may need to rebase and rename this.

Reply:
Yes noted, thanks for the heads up 🙂